Core/HLE/sceKernelMemory.cpp

```diff
@@ -2301,3 +2301,96 @@ void Register_SysMemUserForUser()
 {
 	RegisterModule("SysMemUserForUser", ARRAY_SIZE(SysMemUserForUser), SysMemUserForUser);
 }
+
+
+struct HeapInformation : public KernelObject {
+	HeapInformation() {}
+	int uid;
+	int partitionId;
+	u32 size;
+	int flags;
+	u32 address;
+	std::string name;
+	BlockAllocator alloc;
+	static int GetStaticIDType() { return SCE_KERNEL_TMID_Fpl; }//  wrong
+	int GetIDType() const override { return SCE_KERNEL_TMID_Fpl; }//  wrong
```

Comment:
We don't use the "XyzInformation" convention in PPSSPP, even if JPCSP does.  I really think we should have testing to understand how this differs from sceHeap.  Also, testing can identify which TMID is uses - there's a syscall that gives you the type id for a uid, iirc.

I can't remember for sure, but I thought we used the TMID for savestates, though.  I don't think it's good if they're wrong.

-[Unknown]